eperusteet-opintopolku-app/v2/src/routes/opetussuunnitelmat/RouteOpetussuunnitelma.vue

```diff
@@ -72,7 +72,10 @@ export default class RouteOpetussuunnitelma extends Vue {
     if (this.opetussuunnitelmaDataStore?.opetussuunnitelma?.jotpatyyppi === 'MUU') {
       return 'koulutustyyppi_muu';
     }
-    return this.opetussuunnitelmaDataStore.koulutustyyppi;
+    else if (this.opetussuunnitelmaDataStore.koulutustyyppi) {
+      return this.opetussuunnitelmaDataStore.koulutustyyppi;
+    }
+    return this.tyyppi === 'yhteinen' ? 'koulutustyyppi_1' : null;
```

Comment:
voisko olla mieluummin 
`return this.tyyppi === 'yhteinen' ? 'koulutustyyppi_1' : this.opetussuunnitelmaDataStore.koulutustyyppi`